Bug 2017535: fix broken Argo CD link image

reginapizza opened the pull request:
Description: In the GitOps Environments Details page, the image before the link to Argo CD UI was showing a broken image because the Argo CD operator (the source it was grabbing it from) was not available in the cluster. I have changed it to be a link to the argocd logo in the argocd project, which is unlikely to change. 

Jira Id : https://issues.redhat.com/browse/GITOPS-1500

Screen shots:
![non-broken-argocd-link-image](https://user-images.githubusercontent.com/50851526/138980711-af6bd7d5-d12e-4e17-85d8-c318b283c3f1.png)

andrewballantyne commented on frontend/packages/gitops-plugin/src/components/details/GitOpsDetails.tsx:
```diff
@@ -179,7 +176,7 @@ const GitOpsDetails: React.FC<GitOpsDetailsProps> = ({ envs, appName }) => {
                                     <img
                                       loading="lazy"
-                                      src={ArgoCDIcon}
+                                      src="https://cd.apps.argoproj.io/assets/images/logo.png"
```
As I mentioned in a DM to Regina; if this is not available in offline envs, we probably should look to get another icon.

andrewballantyne commented:
/assign

reginapizza commented:
Ok I've fixed this so it now has a locally stored PNG file. This is what it looks like in console now: 

![argoIcon](https://user-images.githubusercontent.com/50851526/139952213-57a38561-bcb2-4679-9b07-2b0e70b4e223.png) 

(also removed a commented out import I saw)

reginapizza commented:
/bugzilla refresh

andrewballantyne commented:
/retest

reginapizza commented:
/bugzilla refresh

andrewballantyne commented:
/lgtm

reginapizza commented:
/cherry-pick release-4.9
